src/plone/app/multilingual/locales/zh_TW/LC_MESSAGES/plone.app.multilingual.po

```diff
@@ -0,0 +1,678 @@
+#. Default: "Use language codes in URL path for manual override."
+#: ./browser/controlpanel.py:123
+msgid "description_language_codes_in_URL"
+msgstr ""
+
+#. Default: "Go to the user's browser preferred language related folder"
+#: ./browser/menu.py:160
+msgid "description_language_folder"
+msgstr ""
+
+#. Default: "Use the language of the content item."
+#: ./browser/controlpanel.py:114
+msgid "description_language_of_the_content"
+msgstr ""
+
+#. Default: "The migration process is divided into four steps, listed below. Each one of the steps has a well defined purpose explained. Some of them are destructive, so please read each one of it and do not try to run it on production servers without testing it previously."
+#: ./browser/templates/migration.pt:99
+msgid "description_migrate"
+msgstr ""
+
+#. Default: "It can happen that there are relations left that contain LinguaPlone relations. By executing this step, the catalog gets reset and rebuilt. It CAN happen that things go bad. In that case relations get lost. It is therefor important that you have a backup!"
+#: ./browser/templates/cleanup.pt:80
+msgid "description_migrate_after"
+msgstr ""
+
+#. Default: "Here you will see the results of the migration process"
+#: ./browser/templates/migration.pt:265
+msgid "description_migration_results"
+msgstr ""
+
+#. Default: "Move the content that is on the root folder to the default language folder. This value is not stored so you need to check every time you want to run it"
+#: ./browser/controlpanel.py:97
+msgid "description_move_content_to_language_folder"
+msgstr ""
+
+#. Default: "After creating a new translation redirecto to babel view"
+#: ./interfaces.py:31
+msgid "description_redirect_babel_view"
+msgstr ""
+
+#. Default: "Use this field to define portal types that will not be traversed for finding content to move. All content of those types will be moved with all contained sub-items. This is useful for content types that are themselves translatable, but contain sub-content that is not translatable or does not have a language set.<br /> This list is pre-filled with some portal types, but you can add additional ones if needed, one per line."
+#: ./browser/templates/migration.pt:161
+msgid "description_relocate_blacklist"
+msgstr ""
+
+#. Default: "This step will move the site's content to its corresponding root language folder and previously will make a search for misplaced content through the site's content tree and will move them to its nearest translated parent. This step is destructive as it will alter your content tree structure. Make sure you have previously configured your site's languages properly in the 'Site Languages' tab of the 'Languages' control panel."
+#: ./browser/templates/migration.pt:149
+msgid "description_relocate_content"
+msgstr ""
+
+#. Default: "Delete translations or remove the relations"
+#: ./browser/menu.py:143
+msgid "description_remove_translations"
+msgstr "Übersetzungen löschen oder die Verweise entfernen"
```

Comment:
@l34marr there's some German strings remaining in here.  I guess they should be set to empty?